### PR TITLE
LIME-131: fixed swagger docs generation. Updated outdated schemas

### LINKED ISSUE #131: BUG: Fix swagger docs generation. Update outdated schemas
Fix bug with blank page for swagger docs.
Update outdated schemas for all endpoints.

### PR DESCRIPTION
**Pull request by jhladkov**

### CODE REVIEW

**Comment by jhladkov** [comment]
image

File: backend/src/bundles/users/user.controller.ts [hunk 1]
```diff
@@ -23,6 +23,29 @@ import { UsersApiPath } from './enums/enums.js';
...
+ *            type: string
+ *            nullable: true
+ *            format: date
+ *          weight:
+ *            type: number
+ *            nullable: true
+ *          height:
+ *            type: number
+ *            nullable: true
+ *          gender:
```
Comment (dimapopovych) [inline backend/src/bundles/users/user.controller.ts:63]: narrow to enum

File: backend/src/bundles/users/user.controller.ts [hunk 2]
```diff
@@ -42,18 +65,23 @@ class UserController extends BaseController {
 
     /**
      * @swagger
-     * /users:
+     * /api/v1/users/:
```
Comment (dimapopovych) [inline backend/src/bundles/users/user.controller.ts:88]: this is a private endpoint, let's add security section for it

File: backend/src/bundles/auth/auth.controller.ts [hunk 3]
```diff
@@ -62,8 +97,10 @@ class AuthController extends BaseController {
 
     /**
      * @swagger
-     * /auth/sign-up:
+     * /api/v1/auth/sign-up:
```
Comment (dimapopovych) [inline backend/src/bundles/auth/auth.controller.ts:107]: The /api/v1 part of the URL is redundant, other versions will be available at different URLs, such as /v2/documentation etc

**Review by dimapopovych — CHANGES_REQUESTED** [review]

File: backend/src/bundles/auth/auth.controller.ts (hunk 3 above)
Comment (dimapopovych) [reply backend/src/bundles/auth/auth.controller.ts:107]: or maybe not) 
i'm not sure if we will be able to send requests in swagger without this part. double-check pls

Comment (jhladkov) [reply backend/src/bundles/auth/auth.controller.ts:107]: Unfortunately, we won't be able to send request without this part: `/api/v1/`

**Comment by jhladkov** [comment]
image

**Review by nikita-remeslov — APPROVED** [review]

**Review by DavidCharif — APPROVED** [review]

**Review by dimapopovych — APPROVED** [review]

**Pull request merged**